### ISSUE TITLE
final layer's activation function

### ISSUE THREAD

**Issue by babbu3682** [lukemelas/EfficientNet-PyTorch#175]
why did not you use the activation function like sigmoid or softmax in the final layer?
is it okay to use just the logit value and then input to loss function(cross-entropy loss and BCELoss ..)?

**Comment by marrrcin** [comment]
I think it's better to return logits, as it gives more flexibility to the user of the library to adjust the core EfficientNet model to the desired final task - whether it's binary classification, multi-class or something else.

**Comment by lukemelas** [comment]
Yup, mrrrcin is totally right! Note that `nn.CrossEntropyLoss` has `nn.LogSoftmax()` built in: https://pytorch.org/docs/stable/nn.html#crossentropyloss
